mmv1/products/compute/PublicAdvertisedPrefix.yaml

```diff
@@ -51,6 +51,16 @@ examples:
       prefixes_name: 'my-prefix'
     test_env_vars:
       desc: :PAP_DESCRIPTION
+  - !ruby/object:Provider::Terraform::Examples
```

Comment:
could you make the changes to the prefix resources in a separate PR please?

Reply:
I've opened #11406. Will pull the changes out of here shortly.